docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -6,21 +6,23 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guid
 
 # Deployment
 
-Strapi gives you many possible deployment options for your project or application. Strapi can be deployed on traditional hosting servers or services such as 21YunBox, Render, Heroku, AWS, Azure and others. The following documentation covers how to develop locally with Strapi and deploy Strapi with various hosting options.
+Strapi provides many deployment options for your project or application. Your Strapi applications can be deployed on traditional hosting servers or your preferred hosting provider. 
 
-::: strapi Database deployment
-Deploying databases along with Strapi is covered in the [databases guide](/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md#databases-installation-guides).
+The following documentation covers how to develop locally with Strapi and deploy Strapi with several common hosting options.
+
+::: strapi Community Guides
+In addition to the official deployment guides maintained by Strapi that are found here, community maintained guides for additional providers are available in the [Strapi Community Guides](https://forum.strapi.io/c/community-guides/28).
```

Comment:
My punctuation usage isn't great, but should "community maintained" be hyphenated?